### PR TITLE
chore(release): 6.0.0-alpha.0

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Bump to 6.0.0-alpha.0

# Changelog

- bumped versions to 6.0.0-alpha.0
- updated release notes in `app-shell` and `api` with basic information 

# Review requests

-  release notes look reasonable for a first pass?
-  Change-logs look reasonable?

# Risk assessment

low

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
 
-Welcome to the beta release of v5.1.0 of the OT-2 software!
+Welcome to the alpha release of v6.0.0 of the OT-2 software!
 
-In conjunction with changes on the app-side, the 5.1.0 release reshapes the way your OT-2 stores runs and protocols. We're very excited for you to try this new experience out, and we appreciate your feedback (and patience) as you use this new software.
+In conjunction with changes on the app-side, the 6.0.0 release reshapes the way your OT-2 stores runs and protocols. We're very excited for you to try this new experience out, and we appreciate your feedback (and patience) as you use this new software.
 
-This is beta software! You may experience unexpected crashes or missing features not detailed here. Please see the [issue tracker][] for more details and to leave feedback.
+This is alpha software! You may experience unexpected crashes or missing features not detailed here. Please see the [issue tracker][] for more details and to leave feedback.
 [issue tracker]: https://github.com/Opentrons/opentrons/issues/new?assignees=y3rsh%2Cnusrat813&labels=5.1-feedback&template=testing_feedback.yml&title=5.1+Feedback%3A++%3Ctitle%3E
```
Comment (y3rsh) [inline api/release-notes.md]:
```suggestion
[issue tracker]: https://github.com/Opentrons/opentrons/issues/new?assignees=y3rsh%2Cnusrat813&labels=6.0-feedback&template=testing_feedback.yml&title=6.0+Feedback%3A++%3Ctitle%3E
```

**Review by y3rsh — APPROVED** [review]
Saw 1 thing.

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
-For more details about this release, please see the full [technical change
+**For** more details about this release, please see the full [technical change
```
Comment (ecormany) [inline app-shell/build/release-notes.md]:
```suggestion
For more details about this release, please see the full [technical change
```
Not sure why this got bolded.

File: app-shell/build/release-notes.md [hunk 3]
```diff
@@ -25,17 +25,11 @@ This is beta software! You may experience unexpected crashes or missing features
...
 ## Known Issues
 
-This beta is a work in progress! We'll be addressing the following features and issues in upcoming releases of 5.1.0.
-
-- You will not be able to view historical protocol runs on a Device Detail Page
-- You will not be able to Factory Reset
-- You will not be able to specify an alternate path to a local python interpreter
-- Portions of the Protocol Detail page are currently in development
-- Magnetic Module Set Engage Height controls may not function properly
+This alpha is a work in progress! We'll be addressing the following features and issues in upcoming releases of 6.0.0.
```
Comment (ecormany) [inline app-shell/build/release-notes.md:32]: Did we address all 5 bullets from the previous revision? 
- If not, bring over ones that are still unresolved.
- If yes, it's odd to say "the following…" and then have no list. So I'd suggest changing it to "We'll be addressing issues found during alpha testing in upcoming releases of 6.0.0." or similar.

**Review by ecormany — APPROVED** [review]
Happy with the minimal changes to release notes for now. See 2 small changes that I flagged.

File: app-shell/build/release-notes.md (hunk 3 above)
Comment (b-cooper) [reply app-shell/build/release-notes.md:32]: Good call, we'll add contents to this section for the Beta bump

**Review by bndo — APPROVED** [review]
LGTM

**Pull request merged**